docs/architecture/adr-040-storage-and-smt-state-commitments.md

```diff
@@ -123,6 +170,8 @@ We change the storage layout of the state machine, a storage hard fork and netwo
 + Decoupling state from state commitment introduce better engineering opportunities for further optimizations and better storage patterns.
 + Performance improvements.
 + Joining SMT based camp which has wider and proven adoption than IAVL. Example projects which decided on SMT: Ethereum2, Diem (Libra), Trillan, Tezos, LazyLedger.
++ Multistore removal fixes a long standing issues with current MultiStore design.
```

Comment:
```suggestion
+ Multistore removal fixes a longstanding issue with the current MultiStore design.
```

Reply:
It might also be nice to be more specific as to what the longstanding issue is here.

Reply:
added complexity with no benefits.